### PR TITLE
Init express server(to replace nest)

### PR DESCRIPTION
**Pull request by czhen-zilliz**

### CODE REVIEW

File: express/src/interceptors/index.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * Normally depend on status which from milvus service.
+ */
+export const ErrorInterceptor = (
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  console.log("---error interceptor---\n%s", err);
+  if (res.headersSent) {
```
Comment (nameczz) [inline express/src/interceptors/index.ts:73]: what is headersSent?

File: express/src/interceptors/index.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      `${req.method} ${
+        req.originalUrl
+      } [FINISHED] ${durationInMilliseconds.toLocaleString()} ms`
+    );
+  });
+
+  res.on("close", () => {
+    const durationInMilliseconds = getDurationInMilliseconds(start);
+    const { statusCode = "" } = res;
+    console.log(
```
Comment (nameczz) [inline express/src/interceptors/index.ts:52]: Need some special log instead of console.log. Add todo comment .

**Review by nameczz — CHANGES_REQUESTED** [review]

File: express/src/interceptors/index.ts (hunk 1 above)
Comment (czhen-zilliz) [reply express/src/interceptors/index.ts:79]: Boolean property that indicates if the app sent HTTP headers for the response.

Comment (czhen-zilliz) [reply express/src/interceptors/index.ts:79]: Here to prevent sending response after header has been sent

**Review by nameczz — APPROVED** [review]

**Pull request merged**